source/wp-content/themes/wporg-developer-2023/src/code-comment-edit/style.scss

```diff
@@ -14,84 +14,5 @@
 			padding-top: var(--wp--preset--spacing--30);
 		}
 	}
-
```

Comment:
All this CSS is in the [main css](https://github.com/WordPress/wporg-developer/pull/251/files#diff-5a967de023463491bebbfaca0909be4be783855fcace73f96a9a0e9c8060224aR194)

Perhaps it can be scoped to this block... not sure